code/src/nuvla/job/actions/utils/resource_log_fetch.py

```diff
@@ -56,7 +58,12 @@ def all_components(self):
 
     def get_component_logs(self, component: str, since: datetime,
                            lines: int) -> str:
-        return self.connector.log(component, since, lines)
+
+        self.log.info(f"Calling get_component_logs...\n\
+            Connector is set to: {self.connector.__class__.__name__}")
+
+        return self.connector.log(component, since, lines, \
+            namespace=Deployment.uuid(self.deployment))
```

Comment:
This will fail for a NuvlaEdge on Docker because the `log` function on the `NuvlaBox` class doesn't take keyword arguments.

Reply:
This has been removed and put back to original form.